ewcfg.py

```diff
@@ -8095,7 +8097,8 @@ def atf_hooves(ctn=None):
 		role = "Juvie's Row",
 		pvp = False,
 		property_class = property_class_b,
-		community_chest = chest_id_juviesrow
+		community_chest = chest_id_juviesrow,
+		pier_type = fish_slime_saltwater
```

Comment:
wouldn't it make more sense to put this info with the pier pois?

Reply:
pier pois aren't used since we removed subzones, so when a fisher !casts their assigned pier is the mother district

Reply:
i could use `chname_to_poi` to get the pier poi, if you'd prefer it that way

Reply:
why though? what if we want to have multiple piers in the same location?

Reply:
> i could use `chname_to_poi` to get the pier poi, if you'd prefer it that way

yeah, I'd like that